bazel/envoy_select.bzl

```diff
@@ -108,6 +108,13 @@ def envoy_select_hot_restart(xs, repository = ""):
         "//conditions:default": xs,
     })
 
+# Selects the given values if signal trace is enabled in the current build.
+def envoy_select_signal_trace(xs, repository = ""):
+    return select({
+        repository + "//bazel:disable_signal_trace": [],
```

Comment:
Shall we add the mobile-specific logic here too, or do you want to do so in a followup?